velox/common/caching/SsdCache.h

```diff
@@ -24,6 +24,10 @@ namespace facebook::velox::cache {
 #define VELOX_SSD_CACHE_LOG(severity) \
   LOG(severity) << VELOX_SSD_CACHE_LOG_PREFIX
 
+namespace test {
+class SsdCacheTestHelper;
+}
+
 class SsdCache {
```

Comment:
Can you remove all existing testing methods from the two classes with test helper? thanks!

Reply:
Let's do it in a follow up PR?